catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/OverrideAttributesSupport.java

```diff
@@ -43,31 +43,29 @@ public static void overrideAttributes(
       Metacard currentMetacard = metacardMap.get(contentItem.getId());
       Metacard overrideMetacard = contentItem.getMetacard();
 
-      Metacard updatedMetacard = overrideMetacard(currentMetacard, overrideMetacard, false, false);
+      Metacard updatedMetacard = overrideMetacard(currentMetacard, overrideMetacard, false);
 
       metacardMap.put(contentItem.getId(), updatedMetacard);
     }
   }
 
   public static Metacard overrideMetacard(
-      Metacard currentMetacard,
-      Metacard overrideMetacard,
-      boolean ignoreType,
-      boolean onlyFillNull) {
-    MetacardType updatedMetacardType = currentMetacard.getMetacardType();
-    if (!ignoreType && !MetacardImpl.BASIC_METACARD.equals(overrideMetacard.getMetacardType())) {
-      updatedMetacardType = overrideMetacard.getMetacardType();
-    }
+      Metacard currentMetacard, Metacard overrideMetacard, boolean onlyFillNull) {
 
-    Metacard updatedMetacard = new MetacardImpl(currentMetacard, updatedMetacardType);
+    MetacardImpl updatedMetacard = new MetacardImpl(currentMetacard);
```

Comment:
That constructor creates a new internal map and only includes the attributes from the metacard type descriptors. It could work here but there is still the possibility that some attributes don't have descriptors. I know you can't count on attributes making it through without a descriptor but this way we don't loose them here. Was just thinking this was the safest way but could change it if you want